CHANGELOG.md

```diff
@@ -7,7 +7,13 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased]
 
 ### Added
+
+- Add `-version` flag to show node version
+
 ### Fixed
+
+- #1390, Add -version flag
```

Comment:
Don't need to list every issue here, only put things in "Fixed" if it fixes some bug